ERCS/erc-7536.md

```diff
@@ -0,0 +1,175 @@
+
+### Flexible Implementation of Actions
+
+The actions that can be performed are defined in the edition as a uint96 integer. Each bit in the integer determines whether a particular function can be invoked in the contract, with a maximum of 96 functions. The actions can be implemented flexibly depending on the specific use cases. For instance, if the parent token wants to have full control over the child token, the edition, together with the function setup, can permit the parent token holder to invocate a function that transfers the child token to the parent token holder.
+
+Actions may give the child tokens the following characteristics:
+
+- non-transferable: An SBT that is bound to a user's wallet address
+- revokable: The creator has control over the minted copies. This is suitable for NFTs that encapsulate follower relationships, or funtions as some kind of revokable permits
+- extendable: NFT is valid over a duration and requires extension. This is suitable for recurring memberships.
+- updateable: Allows the child token holder to update the tokenUri when the parent token is updated
+- vote: Child token holder can vote if the vote action bit is set
+- external contract actions: An externally deployed contract that implements functions with permissions controlled by the action bits in the contract with the Distributor interface
+
+### External or Internal Implementation of the Validator Interface 
+
+The Validator Interface can be implemented externally as an independent contract, or internally as part of the contract that issues the child token. The former approach is more upgrade-friendly, i.e., validation contracts can be easily swapped to a higher version, while still maintaining compatibility to past versions. More it permits multiple different validators to coexist at the same time. The latter one is less composable, but more secure, as it does not depend on third-party code. This is prefered if the validated rules are unlikely to change in the future.
+
+### Flexible Implementation of Validation Rules
+
+The Validator Contract can be customized to enforce rules, including:
+
+- Fee: Requires payment to mint
+- Free: No Condition to mint
+- NFT Holder: Process a particular NFT to mint
+- [ERC-20](./eip-20.md) Holder: Process a certain amount of ERC-20 tokens to mint
+- Whitelist: In the whitelist to mint.
+- Limited Issuance: Fixed Maximum number of issued copies.
+- Limited Time: Enables minting within a particular time frame.
+
+
+## Backwards Compatibility
+
+This standard is compatible with [ERC-721](./eip-721.md).
+
+## Reference Implementation
+
+The reference implementation is given in  `../assets/erc-7536/`.
```

Comment:
```suggestion
The reference implementation is given in  [`../assets/erc-7536/`](../assets/eip-7536/contracts/Distributor.sol).
```

Reply:
Sure, thanks. We will modify it according to the suggestion. At the same time, we are also revamping it to make it compatible with prior erc.